git-backdate

```diff
@@ -82,32 +91,34 @@ def rewrite_history(
     elif no_business_hours:
         min_hour = 18
         max_hour = 23
-    duration = len(days)
```

Comment:
As `len(days)` doesn't change, isn't it preferable to compute it once outside the loop rather than again and again inside the loop?